### ISSUE TITLE
Doesn't build. 

### ISSUE THREAD

**Issue by shawnjones253** [microsoftgraph/consoleApp-deltaQuery-dotNet-sample#14]
Somebody forgot to make sure this built before leaving it forever.

**Comment by jthake-msft** [comment]
@jasonjoh do you want to have a look at this when you get a chance?

**Comment by jasonjoh** [comment]
Yep, sure will. @shawnjones253 I'm going to try to get to this ASAP, but this is my first time looking at this project :).

**Comment by jasonjoh** [comment]
I suspect the problem here is with Nuget cache path. The projects are trying to load Nuget packages from `..\..\..\..\CxCache\`, but on my machine for example, the default Nuget cache is `.\packages`.

I'm going to submit a PR with a NuGet.config file which should force that path (regardless of machine settings) and fix up the VS project files to use that path. I wish VS had a better method to handle these paths (rather than hard-coding them in the project). :|
